debezium-testing/debezium-testing-testcontainers/src/main/java/io/debezium/testing/testcontainers/MongoDbReplicaSet.java

```diff
@@ -280,7 +280,6 @@ public void start() {
     public void stop() {
         LOGGER.info("[{}] Stopping...", name);
         MoreStartables.deepStopSync(members.stream());
-        network.close();
```

Comment:
@rk3rn3r  Though I likely  aagree with this (the reason for this was that initially network was not configurable and this was likely forgotten) this is exactly the type of change which should have it's own commit clarifying what the isue was here.

Reply:
For me it was just wrong that a single component (of a test stack) closes a network meant for testing a whole stack. I didn't feel this needs discussion, a separate task or commit.

Reply:
Yes for you and for me it makes sense as well (since I know what is going on). The prupose is for somebody else to be able to track why this changed

Reply:
I split the PR in 3 commits now to reflect and isolate the bugfix.